pkg/out/out.go

```diff
@@ -35,3 +37,8 @@ func FormatExampleCommands(command string, examples []Example) (res string) {
 	}
 	return
 }
+
+func Long(in string) string {
```

Comment:
can you add a comment what this is doing?

Reply:
done